feature/platform/controllercard/tests/README.md

```diff
@@ -0,0 +1,114 @@
+# gNMI-1.15 Controller Card redundnacy test
+
+## Summary
+- collect inventory data for each controller card
+- Verify last restart time is updated
+
+## Procedure
+
+### test 1 Contyroller Card inventory
+
+> NOTE: Theis test is practically identical to Fabric part of
+> https://github.com/openconfig/featureprofiles/tree/main/feature/platform/tests/telemetry_inventory_test;
+> Hence code could be reused and then removed form telemetry_inventory_test.
+
+* collect following attributes for each component of CONTROLLER_CARD type and verify corectness (mostly non-empty string)
+  *   /components/component/state/empty
+  *   /components/component/state/location
+  *   /components/component/state/oper-status
+  *   /components/component/state/switchover-ready
+  *   /components/component/state/redundant-role
+  *   /components/component/state/last-switchover-time
+  *   /components/component/state/last-switchover-reason/trigger
+  *   /components/component/state/last-switchover-reason/details
+  *   /components/component/state/last-reboot-time
+  *   /components/component/state/last-reboot-reason
+ 
+  *   /components/component/state/description             
+  *   /components/component/state/hardware-version
+  *   /components/component/state/id
+  *   /components/component/state/mfg-name
+  *   /components/component/state/name
+  *   /components/component/state/parent
+  *   /components/component/state/part-no
+  *   /components/component/state/serial-no
+  *   /components/component/state/type
+
+* store list of present components of CONTROLLER_CARD type
+
+### test 2 switchover
+* Verify that all controller_cards has `switchover-ready=TRUE`
+* Collect and store `redundant-role` for each controller_card as "previous-role"
+* Initiate controller-card switchover
+* Wait 15 seconds
+* Collect `redundant-role` for each controller_card. Compare it wit "previous-role"
+  * for controller_card of **current** "PRIMARY" role, **previous** role must be "SECONDARY"
+  * for at least one controller_card of **previous** role "SECONDARY" role, **current** role must be "PRIMARY"
+  * shall more then 2 controller_cards exist, their previour and current states could be equal and be "SECONDARY"
+* Until (`switchover-ready=TRUE` on all controller_cards OR `last-switchover-time` is moret then 20min ago)
+  * Wait(5min)
+  * Verify that all controller_cards has `switchover-ready=TRUE`; if so test PASSED
+
+### test 2 Redundancy
```

Comment:
For this test:
1) Is this a negative test? I think people in prod will first do a switchover and then shut the controller card down. Do we want to follow that? Do we want to add one more test and keep this one?
2) Please update typos and maybe slightly rephrase, as it is hard to read